config/products-validation.ts

```diff
@@ -0,0 +1,15 @@
+import * as Joi from "joi";
+
+export const addProductSchema = Joi.object({
```

Comment:
think about a way to use only one joi schema for both create and update validations